src/test/scala/unit/inference/SQLInferenceDataStoreSpec.scala

```diff
@@ -586,6 +586,120 @@ trait SQLInferenceRunnerSpec extends FunSpec with BeforeAndAfter { this: SQLInfe
           .map(rs => rs.long("count")).single.apply().get
         assert(numVariablesInc === 20)
       }
+
+      it("should work with materialization and incremental modes (multinomial)") {
+        cancelUnlessPostgres()
+        // XXX Incremental workflow not supported on Postgres-XL as inference.PostgresInferenceRunner.groundVariables raises the following error:
+        //   org.postgresql.util.PSQLException: ERROR: could not plan this distributed update
+        //   Detail: correlated UPDATE or updating distribution column currently not supported in Postgres-XL.
+        if (dataStoreHelper.isUsingPostgresXL) pending
+        // XXX Incremental workflow not supported on Greenplum as datastore.PostgresDataStore.createTableIfNotExists raising following error:
```

Comment:
These lines were in master, so I used it here. I think incremental has been fixed for greenplum and postgresxl.